File: src/SOP8/Base/Account/Base.Account.IBLL/Response/ResponseLoginKey.cs
using Response;

namespace Base.Account.IBLL.Response
{
    public class ResponseLoginKey : MessageResult
    {
        private string m_strLoginKey = "";
        private string m_strSalt = "";
        private bool m_externalLogin = false;

        public string LoginKey
        {
            get { return m_strLoginKey; }
            set { m_strLoginKey = value; }
        }

        public string Salt
        {
            get { return m_strSalt; }
            set { m_strSalt = value; }
        }

        public bool ExternalLogin
        {
            get { return m_externalLogin; }
            set { m_externalLogin = value; }
        }
    }
}

File: src/SOP8/Egovframework/Base/base-account-ibll/src/main/java/egovframework/base/account/ibll/response/ResponseLoginKey.java
package egovframework.base.account.ibll.response;

import egovframework.base.response.MessageResult;

public class ResponseLoginKey extends MessageResult {

    private String loginKey = "";
    private String salt = "";
    private boolean externalLogin = false;

    public String getLoginKey() {
        return loginKey;
    }

    public void setLoginKey(String loginKey) {
        this.loginKey = loginKey;
    }

    public String getSalt() {
        return salt;
    }

    public void setSalt(String salt) {
        this.salt = salt;
    }

    public boolean isExternalLogin() {
        return externalLogin;
    }

    public void setExternalLogin(boolean externalLogin) {
        this.externalLogin = externalLogin;
    }
}